lib/osf-components/addon/components/resources-list/edit-resource/component.ts

```diff
@@ -38,11 +38,19 @@ export default class EditResourceModal extends Component<Args> {
             presence: true,
             translationArgs: { description: this.intl.t('osf-components.resources-list.edit_resource.doi') },
         })],
-        resourceType: [validatePresence({
-            type: 'blank',
-            presence: true,
-            translationArgs: { description: this.intl.t('osf-components.resources-list.edit_resource.output_type') },
-        })],
+        resourceType: [
+            validatePresence({
+                type: 'blank',
+                presence: true,
+                translationArgs: {
+                    description: this.intl.t('osf-components.resources-list.edit_resource.output_type'),
+                },
+            }),
```

Comment:
I wonder if we need the Presence validation here. With how things are being sent from the API, we should always have a value for this field.